pkg/settlement/swap/chequebook/common_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package chequebook_test
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethersphere/bee/pkg/settlement/swap/chequebook"
+	"github.com/ethersphere/sw3-bindings/v2/simpleswapfactory"
+)
+
+type backendMock struct {
```

Comment:
It would be good to panic on all not implemented methods for this mock, just to make sure that nobody relies on them in the future, until implemented.

Reply:
Done.